src/forms/create_ticket.ts

```diff
@@ -22,7 +22,7 @@ const omitFields = ['Group', 'Status'];
 export async function newCreateTicketForm(call: AppCallRequest): Promise<AppForm> {
     const context = call.context as CtxExpandedBotAdminActingUserOauth2User;
     const zdOptions: ZDClientOptions = {
-        oauth2UserAccessToken: context.oauth2.user.access_token,
+        oauth2UserAccessToken: context.oauth2.user.token.access_token,
```

Comment:
It's not possible because the user cannot open the create modal unless they are connected with checks for the token

Reply:
I don't think we should have cross-endpoint assumptions though. This can be an arbitrary http request coming to the app. 2/5 we should at least have a ticket to make sure each endpoint's assumptions are verified by that same endpoint.

Reply:
ticket created https://mattermost.atlassian.net/browse/MM-35440